common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -144,20 +144,6 @@ public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTic
                         overlay.getWidth(),
                         overlay.getHeight());
 
-                int xOffset =
```

Comment:
Actually I think this part was good. (If I understand the code correctly)

While the alignment now works correctly, it will align the text with the *outside* of the box, making the text overwrite the box border. It would look better if it were aligned with the *inside* of the box (and possibly with a one pixel margin). I think this code tried to achieve that. (Even though it's odd that the additional offset was different in different directions (3 vs 5)).

Reply:
It caused weirdness with small overlays. Maybe we can just have an universal offset. I will try that in a second.